radixdlt/src/integration/java/com/radixdlt/consensus/simulation/BFTSimulatedTest.java

```diff
@@ -160,26 +166,14 @@ public static Builder builder() {
 		return new Builder();
 	}
 
-	/**
-	 * Runs the test for a given time. Returns either once the duration has passed or if a check has failed.
-	 * Returns a map from the check name to the result.
-	 *
-	 * @param duration duration to run test for
-	 * @param timeUnit time unit of duration
-	 * @return map of check results
-	 */
-	public Map<String, Optional<BFTCheckError>> run(long duration, TimeUnit timeUnit) {
-		TestEventCoordinatorNetwork network = TestEventCoordinatorNetwork.builder()
-			.latencyProvider(this.latencyProvider)
-			.build();
-		SimulatedBFTNetwork bftNetwork =  new SimulatedBFTNetwork(nodes, network, pacemakerTimeout, getVerticesRPCEnabled);
+	private Observable<Pair<String, Optional<BFTCheckError>>> runChecks(RunningNetwork runningNetwork, long duration, TimeUnit timeUnit) {
 		List<Pair<String, Observable<Pair<String, BFTCheckError>>>> assertions = this.checks.keySet().stream()
 			.map(name -> {
 				BFTCheck check = this.checks.get(name);
```

Comment:
I think `BFTCheck` ought to be renamed, it is an "actor", some entity performing something, by calling invoking the method `check`. Thus `BFTChecker checker` reads out better and then `checker.check` reads out much better than `check.check`. 

But why not name it `BFTConsensusAsserter` and method `assert`, which is what the documentation of the method implies and `assert` is a much better word - since it is more specific - than `check` which does not carry the same meaning.

Reply:
`assert` is a language keyword, so we can't use that